openslides/users/static/templates/users/user-detail.html

```diff
@@ -1,2 +1,30 @@
-<h1>Persönliche Daten</h1>
-{{ user.get_short_name() }}
+<h1>{{ user.get_short_name() }}</h1>
+
+<div id="submenu">
+    <a ui-sref="users.user.list" class="btn btn-sm btn-default">
+        <i class="fa fa-angle-double-left fa-lg"></i>
+        Back to overview
+    </a>
+</div>
+
+<div class="user_details form-group">
+    <fieldset>
+        <legend>Personal data</legend>
```

Comment:
You probably want to translate this (also the string in line 6 and other strings in this file)